packages/wallet/src/redux/channel-state/reducer.ts

```diff
@@ -29,6 +29,7 @@ import { StateWithSideEffects } from '../utils';
 import { ethers } from 'ethers';
 import { channelID } from 'fmg-core/lib/channel';
 import { WalletAction, COMMITMENT_RECEIVED } from '../actions';
+import { Commitment } from 'fmg-core/lib/commitment';
```

Comment:
shouldn't this be imported from `fmg-core` itself?

Reply:
Yes, for some reason vscode auto import seems to want to pull it off lib.

Reply:
Also it looks like an issue we have in a lot of places. We should probably go through and clean it up at some point.